reconplogger.py

```diff
@@ -178,13 +187,26 @@ def logger_setup(logger_name='plain_logger', config=None, env_prefix='LOGGER', i
         raise ValueError('env_prefix is required to be a non-empty string.')
     env_cfg = env_prefix + '_CFG'
     env_name = env_prefix + '_NAME'
+    env_level = env_prefix + '_LEVEL'
 
     # Configure logging
     load_config(os.getenv(env_cfg, config))
 
     # Get logger
     logger = get_logger(os.getenv(env_name, logger_name))
 
+    # Override log level if set
```

Comment:
@nischalhp yes, the priority of the env var should be higher. But it is like this in the code. If env level is set, then `level` is replaced with whatever value is in the env level. Then `level` is used to set the logger level.

Reply:
aah yes, I see that. Good stuff. I did not see that the method parameter and the variable being used inside the function are the same. Just FYI, its frowned upon to do that according to pep-8 guidelines as it might lead to some confusions while reading code, but it looks good. Approving the request.